docs/code-quality/ca1002.md

```diff
@@ -49,6 +49,16 @@ To fix a violation of this rule, change the <xref:System.Collections.Generic.Lis
 
 Do not suppress a warning from this rule unless the assembly that raises this warning is not meant to be a reusable library. For example, it would be safe to suppress this warning in a performance-tuned application where a performance benefit was gained from the use of generic lists.
 
+## Configurability
+
+If you're running this rule from [FxCop analyzers](install-fxcop-analyzers.md) (and not with legacy analysis), you can configure which parts of your codebase to run this rule on, based on their accessibility. For example, to specify that the rule should run only against the non-public API surface, add the following key-value pair to an .editorconfig file in your project:
```

Comment:
Its fine to mention `FxCop analyzers` in this PR for consistency with other rule documentations, but just an FYI that I am going to remove reference to a specific package (FxCop, FxCop analyzers, etc.) from all rule documentations. At that point, CA rules apply to just source analysis, regardless of the shipping vehicle (NuGet package vs inside .NET SDK itself).

Reply:
Ok! Let me know if I can help.